trade-finance-manager-ui/server/controllers/case/underwriting/pricing-and-risk/index.test.js

```diff
@@ -170,27 +163,26 @@ describe('POST underwriting - pricing and risk edit', () => {
               order: '1',
             },
           },
-          summary: [{
-            text: 'Enter a credit rating',
-            href: '#exporterCreditRating',
-          }],
+          summary: [
+            {
+              text: 'Enter a credit rating',
+              href: '#exporterCreditRating',
+            },
+          ],
```

Comment:
I'm not going to keep commenting this but most of these formatting changes are done by the linter